Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -475,15 +489,33 @@ void Renderer::UpdateDrawRectangle(int backbuffer_width, int backbuffer_height)
 	}
 
 	// Check for force-settings and override.
-	if (g_ActiveConfig.iAspectRatio == ASPECT_FORCE_16_9)
+	switch (g_ActiveConfig.iAspectRatio)
+		{
+	case ASPECT_FORCE_16_9:
 		use16_9 = true;
-	else if (g_ActiveConfig.iAspectRatio == ASPECT_FORCE_4_3)
+		break;
+	case ASPECT_ANALOG_WIDE:
+		use16_9 = true;
+		break;
+	case ASPECT_FORCE_4_3:
 		use16_9 = false;
-
+		break;
+	case ASPECT_ANALOG:
+		use16_9 = false;
+		break;
+	default:
+		use16_9 = g_aspect_wide;
+		break;
+	}
 	if (g_ActiveConfig.iAspectRatio != ASPECT_STRETCH)
 	{
 		// The rendering window aspect ratio as a proportion of the 4:3 or 16:9 ratio
 		float Ratio = (WinWidth / WinHeight) / (!use16_9 ? (4.0f / 3.0f) : (16.0f / 9.0f));
+		if (g_ActiveConfig.iAspectRatio == ASPECT_ANALOG || g_ActiveConfig.iAspectRatio == ASPECT_ANALOG_WIDE)
+		{
+			Ratio = (WinWidth / WinHeight) / VideoInterface::GetAspectRatio(use16_9);
+		}
+		
 		// Check if height or width is the limiting factor. If ratio > 1 the picture is too wide and have to limit the width.
 		if (Ratio > 1.0f)
```

Comment:
I'm not exactly sure what you want you want me to do here.

The existing cropping goes 4:3 -> 5:4 and 16:9->16:10.  If the goal is to have the new settings replace the old "force aspect ratio" settings, then the functionality of the cropping needs to be changed.  I'd suggest cropping "Analog TV" to 4:3 and "Analog TV Wide" to 16:9.